modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -139,3 +139,34 @@ function az_global_footer_update_1020703() {
 
   return t('Updated %count menu link(s) from A-Z Index to Directory in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update Twitter link(s) to new X link(s)
+ *
+ * Twitter icon(s) in the global footer will be updated with
+ * new X icon(s).
+ */
+function az_global_footer_update_1020801() {
+  $updated_count = 0;
+  $menu_link_content_ids = \Drupal::entityQuery('menu_link_content')
+    ->accessCheck(FALSE)
+    ->condition('menu_name', 'az-footer-resources')
+    ->condition('title', 'X')
+    ->condition('link__uri', 'https://x.com/uarizona')
+    ->execute();
+
+  foreach ($menu_link_content_ids as $id) {
+    /** @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content */
+    $menu_link_content = MenuLinkContent::load($id);
+    if ($menu_link_content !== NULL) {
+      // phpcs:disable
+      /* @phpstan-ignore-next-line */
+      $menu_link_content->link->uri = 'https://x.com/uarizona';
```

Comment:
I believe the function needs to match the correct link to update and then make the changes to the title, the URI, and the icon classes. **This code needs to be tested before merging.**

```suggestion
    ->condition('menu_name', 'az-footer-social-media')
    ->condition('title', 'Twitter')
    ->condition('link__uri', 'https://twitter.com/uarizona')
    ->execute();

  foreach ($menu_link_content_ids as $id) {
    /** @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content */
    $menu_link_content = MenuLinkContent::load($id);
    if ($menu_link_content !== NULL) {
      // phpcs:disable
      /* @phpstan-ignore-next-line */
      $menu_link_content->title = 'X, formerly Twitter';
      /* @phpstan-ignore-next-line */
      $menu_link_content->link->uri = 'https://x.com/uarizona';
      /* @phpstan-ignore-next-line */
      $menu_link_content->link->options->attributes->class = 'az-icon-twitter az-icon-x-twitter az-icon-spacing';
```

Reply:
Great catch @bberndt-uaz

Reply:
I'll merge and apply to a multidev